src/motion/basic_target_follower_node.cpp

```diff
@@ -0,0 +1,18 @@
+#include "basic_target_follower.h"
+#include <ros/ros.h>
+#include <string>
+
+int main(int argc, char **argv) {
+  ros::init(argc, argv, "targetfollower");
+  std::string target_topic;
+  if (!ros::param::get("target_topic", target_topic)) {
+    throw ros::Exception("Must specify target_topic parameter");
+  }
+  ros::NodeHandle n;
```

Comment:
Change the nodeHandle variable name from `n` to `nh`. 
That seems to be the convention.